pyperf/__init__.py

```diff
@@ -18,3 +18,12 @@
 
 from pyperf._runner import Runner   # noqa
 __all__.append('Runner')
+
+import sys
+
+# Reset the stats collection if running a --enable-pystats build
+try:
+    sys._stats_off()
+    sys._stats_clear()
```

Comment:
Please don't execute code on "import pyperf". It should be a deliberate action to clear such cache.

Reply:
I can probably get away with putting this in the `Runner` constructor -- it's possible, I suppose, to have more than one of those, but in practice that doesn't happen in the pyperformance benchmarks.

And to be clear, this doesn't affect anything on disk -- it is only the in-memory statistics collected so far.  (This is basically to remove the effect of the statistics collected during pyperf's startup and book-keeping).